Makefile.am

```diff
@@ -36,18 +40,16 @@ bin_PROGRAMS = ctags
 noinst_LIBRARIES = libctags.a
 
 noinst_PROGRAMS =
-noinst_PROGRAMS += packcc
 noinst_PROGRAMS += mini-geany
 
-packcc_CPPFLAGS =
-packcc_CFLAGS  =
-packcc_CFLAGS += $(EXTRA_CFLAGS)
-dist_packcc_SOURCES = $(PACKCC_SRCS)
-
-if HAVE_STRNLEN
-packcc_CPPFLAGS += -DUSE_SYSTEM_STRNLEN
+if HAVE_STRNLEN_FOR_BUILD
+EXTRA_CPPFLAGS_FOR_BUILD = -DUSE_SYSTEM_STRNLEN
 endif
 
+PACKCC = $(top_builddir)/packcc$(BUILD_EXEEXT)
+
+$(PACKCC):
+	$(CC_FOR_BUILD) $(CFLAGS_FOR_BUILD) $(CPPFLAGS_FOR_BUILD) $(EXTRA_CPPFLAGS_FOR_BUILD) $(LDFLAGS_FOR_BUILD) -o $@ $(top_srcdir)/misc/packcc/packcc.c
```

Comment:
build packcc don't follow [The Uniform Naming Scheme](https://www.gnu.org/software/automake/manual/html_node/Uniform.html). because automake auto append `$(EXEEXT)` to `*_PROGRAMS` when generating the Makefile, but here we want to use `$(BUILD_EXEEXT)`.